### DISCUSSION TITLE
After update symfony/psr-http-message-bridge php throws error in getPayload method

### DISCUSSION THREAD

**Discussion by rtcoder** [symfony/symfony#51101] [Q&A] [answered]
Hi, I just install my Laravel dependencies and at the start I got following error on oauth/token request.
I did my own research and it is caused by last `symfony/psr-http-message-bridge` release - exactly this commit https://github.com/symfony/psr-http-message-bridge/commit/6b2f5df6f380681e5910b41a5b824ec6e65dd066
I have no idea why is that. Any idea?

ERROR: Symfony\Component\HttpFoundation\Request::getPayload(): Return value must be of type Symfony\Component\HttpFoundation\InputBag, Symfony\Component\HttpFoundation\ParameterBag returned {"exception":"[object] (TypeError(code: 0): Symfony\\Component\\HttpFoundation\\Request::getPayload(): Return value must be of type Symfony\\Component\\HttpFoundation\\InputBag, Symfony\\Component\\HttpFoundation\\ParameterBag returned at .../vendor/symfony/http-foundation/Request.php:1517)

[stacktrace]
#0 .../vendor/symfony/psr-http-message-bridge/Factory/PsrHttpFactory.php(84): Symfony\\Component\\HttpFoundation\\Request->getPayload()
#1 .../vendor/laravel/framework/src/Illuminate/Routing/RoutingServiceProvider.php(141): Symfony\\Bridge\\PsrHttpMessage\\Factory\\PsrHttpFactory->createRequest(Object(Illuminate\\Http\\Request))
...
#47 {main}
"}

**Comment by Jbustamante6** [comment]
I had the same error on laravel, i couldn't resolve that, i only have blocked the composer.json with v2.2

**Comment by HypeMC** [comment, marked as answer]
Looks like a bug on Laravels side, see https://github.com/laravel/framework/pull/47838
